software/star/align/meta.yml

```diff
@@ -0,0 +1,77 @@
+name: star_align
+description: Align reads to a reference genome using STAR
+keywords:
+  - align
+  - fasta
+  - genome
+  - reference
+tools:
+  - star:
+      description: |
+        STAR is a software package for mapping DNA sequences against
+        a large reference genome, such as the human genome.
+      homepage: https://github.com/alexdobin/STAR
+      manual: https://github.com/alexdobin/STAR/blob/master/doc/STARmanual.pdf
+      doi: 10.1093/bioinformatics/bts635
+params:
+  - outdir:
+      type: string
+      description: |
+        The pipeline's output directory. By default, the module will
+        output files into `$params.outdir/<SOFTWARE>`
+  - publish_dir_mode:
+      type: string
+      description: |
+        Value for the Nextflow `publishDir` mode parameter.
+        Available: symlink, rellink, link, copy, copyNoFollow, move.
+  - enable_conda:
+      type: boolean
+      description: |
+        Run the module with Conda using the software specified
+        via the `conda` directive
+  - singularity_pull_docker_container:
+      type: boolean
+      description: |
+        Instead of directly downloading Singularity images for use with Singularity,
+        force the workflow to pull and convert Docker containers instead.
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - index:
+      type: directory
+      description: STAR genome index
+      pattern: "star"
+output:
+  - bam:
+      type: file
+      description: Output BAM file containing read alignments
+      pattern: "*.{bam}"
+  - log_final:
+      type: file
+      description: STAR final log file
+      pattern: "*Log.final.out"
+  - log_out:
+      type: file
+      description: STAR lot out file
+      pattern: "*Log.out"
+  - log_progress:
+      type: file
+      description: STAR log progress file
+      pattern: "*Log.progress.out"
+  - version:
+      type: file
+      description: File containing software version
+      pattern: "*.{version.txt}"
+
```

Comment:
Need to add the optional outputs here too.

Reply:
Alright :)